ports/libmagic/portfile.cmake

```
@@ -31,9 +32,25 @@ if(VCPKG_TARGET_IS_WINDOWS)
     set(VCPKG_CXX_FLAGS "${VCPKG_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_WARNINGS")
 endif()
 
+set(CONFIG_OPTIONS)
+
+foreach(feature zlib bzlib xzlib zstdlib)
```

```suggestion
set(FEATURE_OPTIONS)

foreach(feature zlib bzlib xzlib zstdlib)
```
Are these all features changes?

---

Renamed.